esphome/components/mcp9808/mcp9808.cpp

```diff
@@ -0,0 +1,81 @@
+#include "mcp9808.h"
+#include "esphome/core/log.h"
+
+namespace esphome {
+namespace mcp9808 {
+
+static const uint8_t MCP9808_REG_AMBIENT_TEMP = 0x05;
+static const uint8_t MCP9808_REG_MANUF_ID = 0x06;
+static const uint8_t MCP9808_REG_DEVICE_ID = 0x07;
+
+static const uint16_t MCP9808_MANUF_ID = 0x0054;
+static const uint16_t MCP9808_DEV_ID = 0x0400;
+
+static const uint8_t MCP9808_AMBIENT_CLEAR_FLAGS = 0x1F;
+static const uint8_t MCP9808_AMBIENT_CLEAR_SIGN = 0x0F;
+static const uint8_t MCP9808_AMBIENT_TEMP_NEGATIVE = 0x10;
+
+static const char *TAG = "mcp9808";
+
+void MCP9808Sensor::setup() {
+  ESP_LOGI(TAG, "Setting up %s...", this->name_.c_str());
```

Comment:
```suggestion
  ESP_LOGCONFIG(TAG, "Setting up %s...", this->name_.c_str());
```
Actually this one is config, -- sorry for the back and forth